.github/workflows/build_and_test.yml

```diff
@@ -52,6 +53,9 @@ jobs:
     strategy:
       matrix:
         include:
+          - image: ubuntu2404
+            python: python3
```

Comment:
now that everybody use python: python3 maybe we could drop it no ?

Reply:
removed all.